src/specs/api.yaml

```diff
@@ -0,0 +1,182 @@
+---
+openapi: 3.0.3
+info:
+  title: "SIS External – Stored Identity"
+  description: "External API for retrieving a user’s stored identity record"
+  version: "1.0.0"
+paths:
+  /user-identity:
+    get:
+      summary: "Retrieve the user’s stored identity record"
+      description: >
+        Returns the signed Stored Identity JWT and associated metadata  
+        indicating whether the record is still valid, whether it has expired,  
+        and the stored vector of trust (VOT).
+      security:
+        - bearerAuth: []
+      tags:
+        - Stored Identity
+      parameters:
+        - $ref: "#/components/parameters/vtr"
+      responses:
+        '200':
+          description: "Stored identity record returned"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/storedIdentityResponse"
+        '401':
+          description: "Unauthorized – e.g. missing or invalid access token"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/oauthError"
+        '403':
+          description: "Forbidden – e.g. expired access token"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/oauthError"
+        '404':
+          description: "Not Found – no stored identity exists for this user"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/oauthError"
+        '500':
+          description: "Internal server error"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/oauthError"
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+  schemas:
+    vtr:
+      description: |-
+        The Level of Confidence that has been attained by the VC bundle.
+        This is used to determine the type of SI that should be checked, either a GPG45 or an Operational Profile
```

Comment:
we no longer support any profiles other than GPG45, but is this in case we do in future?